tests/integration/components/session-card-test.js

```diff
@@ -7,7 +7,7 @@ import { render } from '@ember/test-helpers';
 module('Integration | Component | session card', function(hooks) {
   setupIntegrationTest(hooks);
 
-  const session = EmberObject.create({ id: 123, title: 'Super cool JS', state: 'rejected', event: 'OS Tech', startsAt: new Date(), endsAt: new Date() });
+  const session = EmberObject.create({ id: 123, title: 'Super cool JS', state: 'rejected', event: {name: 'OS Tech', id: 123}, startsAt: new Date(), endsAt: new Date() });
```

Comment:
Codacy found an issue: [A space is required after '{'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5906638)